lib/salus/scanners/osv/maven_osv.rb

```diff
@@ -0,0 +1,150 @@
+require 'salus/scanners/osv/base'
+
+module Salus::Scanners::OSV
+  class MavenOSV < Base
+    class SemVersion < Gem::Version; end
+
+    EMPTY_STRING = "Not Found".freeze
+    DEFAULT_SOURCE = "https://osv.dev/list".freeze
+    DEFAULT_SEVERITY = "MODERATE".freeze
```

Comment:
Does OSV have "MEDIUM" as a severity?  If so, could you set the default to "MEDIUM", just to be consistent with our other tools?

Reply:
Severity labels are - CRITICAL, HIGH, MODERATE, LOW

Reply:
Ok, then we can leave it as MODERATE for now.